src/NServiceBus.RavenDB.Optimistic.AcceptanceTests/NServiceBus.RavenDB.Optimistic.AcceptanceTests.csproj

```diff
@@ -4,31 +4,51 @@
     <TargetFrameworks>net472;netcoreapp3.1;net5.0</TargetFrameworks>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <RootNamespace>NServiceBus.RavenDB.AcceptanceTests</RootNamespace>
-    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\NServiceBus.RavenDB\NServiceBus.RavenDB.csproj" />
+    <ProjectReference Include="..\NServiceBus.RavenDB\NServiceBus.RavenDB.csproj"/>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="GitHubActionsTestLogger" Version="1.2.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-alpha.1900" />
-    <PackageReference Include="NUnit" Version="3.13.2" />
-    <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
-    <PackageReference Include="RavenDB.Client" Version="5.3.0" />
+    <PackageReference Include="GitHubActionsTestLogger" Version="1.2.0"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0"/>
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-alpha.1900"/>
+    <PackageReference Include="NUnit" Version="3.13.2"/>
+    <PackageReference Include="NUnit3TestAdapter" Version="4.1.0"/>
+    <PackageReference Include="RavenDB.Client" Version="5.3.0"/>
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\NServiceBus.RavenDB.AcceptanceTests\**\*.cs" Exclude="**\obj\**" />
+    <Compile Include="..\NServiceBus.RavenDB.AcceptanceTests\**\*.cs" Exclude="**\obj\**"/>
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Remove="\**\ConfigureEndpointInMemoryPersistence.cs"/>
+    <Compile Remove="\**\ConfigureEndpointLearningPersistence.cs"/>
+    <Compile Remove="\**\ConventionEnforcementTests.cs"/>
+    <Compile Remove="\**\DeterministicGuid.cs"/>
+    <Compile Remove="\**\Audit\*.*"/>
+    <Compile Remove="\**\Correlation\*.*"/>
+    <Compile Remove="\**\DataBus\*.*"/>
+    <Compile Remove="\**\DelayedDelivery\*.*"/>
+    <Compile Remove="\**\Forwarding\*.*"/>
+    <Compile Remove="\**\MessageId\*.*"/>
+    <Compile Remove="\**\Pipeline\*.*"/>
+    <Compile Remove="\**\Recoverability\*.*"/>
+    <Compile Remove="\**\Routing\NativePublishSubscribe\*.*"/>
+    <Compile Remove="\**\Satellites\*.*"/>
+    <Compile Remove="\**\Scheduling\*.*"/>
+    <Compile Remove="\**\SelfVerification\*.*"/>
+    <Compile Remove="\**\Serialization\*.*"/>
+    <Compile Remove="\**\TimeToBeReceived\*.*"/>
+    <Compile Remove="\**\Tx\**\*.*"/>
+    <Compile Remove="\**\Versioning\*.*"/>
```

Comment:
What's the reason for removing all these tests?

Reply:
They are actually not relevant for the persistence, and thus we can save us the time of executing those. Although surprisingly, it did not do such a big effect as I was hoping it should have, I still think excluding not relevant tests from the execution makes sense.